src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
```

Comment:
Consider moving the instantiation of 'Random' to a class-level field to avoid creating a new instance every time the 'getRandomBall' method is called.